site/app/views/NavigationView.php

```diff
@@ -252,7 +253,33 @@ public function showGradeables($sections_to_list) {
                     $gradeable_title = '<label>'.$g_data->getName().'</label><a class="external" href="'.$g_data->getInstructionsURL().'" target="_blank"><i style="margin-left: 10px;" class="fa fa-external-link"></i></a>';
                 }
                 else{
-                    $gradeable_title = '<label>'.$g_data->getName().'</label>';
+                    if ($g_data->getType() == GradeableType::ELECTRONIC_FILE) {
+                        $no_teams_flag=1;
```

Comment:
why not use true & false?  That might be more clear to future code maintainers.  

In general, this code logic should have comments documenting the checks, and the reason for the checks.